CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
 
 ---
 
+## [3.2.2] - 2022-10-19
```

Comment:
we haven't publish the 3.2.1 yet

Reply:
packages.json on main is at 3.2.1 !

Reply:
yes we have just prepared the changelog this morning

Reply:
Do we need to publish 3.2.1 ? why not go direct to 3.2.2 ? Or you want me to change to 3.2.1 ?

Reply:
yes please, use the 3.2.1